### PR TITLE
github: set repo & namespace when forking

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
> * [ogr-tests-rpm ](https://softwarefactory-project.io/zuul/t/local/build/85a450e275364331af59fbc22e951066) : FAILURE in 7m 24s
> 
> * [ogr-tests-pip-deps ](https://softwarefactory-project.io/zuul/t/local/build/e76e7f71f7934acf88a364dbb0212986) : FAILURE in 7m 23s

Try to regenerate the response file for the failed test.

File: ogr/services/github/project.py [hunk 1]
```diff
@@ -359,8 +359,13 @@ def fork_create(self) -> "GithubProject":
         :return: fork GithubProject instance
         """
         gh_user = self.github_instance.get_user()
-        fork = gh_user.create_fork(self.github_repo)
-        return GithubProject("", self.service, "", github_repo=fork)
+        fork: Repository = gh_user.create_fork(self.github_repo)
+        project = GithubProject("", self.service, "", github_repo=fork)
```
Comment (lachmanfrantisek) [inline ogr/services/github/project.py]: Can we use `None` instead of `""`? And ideally, keyword arguments so it's clear what's what?

Comment (TomasTomecek) [reply ogr/services/github/project.py]: done

File: tests/integration/test_gitlab.py [hunk 2]
```diff
@@ -24,8 +24,10 @@ def setUp(self):
         persistent_data_file = os.path.join(
             PERSISTENT_DATA_PREFIX, f"test_gitlab_data_{test_name}.yaml"
         )
-        PersistentObjectStorage().storage_file = persistent_data_file
-        if PersistentObjectStorage().mode == StorageMode.write and not self.token:
+        self.pos = PersistentObjectStorage()
```
Comment (lachmanfrantisek) [inline tests/integration/test_gitlab.py]: It does not make sense since it's a singleton. (We need to access that from anywhere.)

**Pull request merged**